cmd/opampsupervisor/README.md

```diff
@@ -22,3 +22,49 @@ The supervisor is currently undergoing heavy development and is not ready for an
    ```
 
 4. The supervisor should connect to the OpAMP server and start a Collector instance.
+
+## Status
+
+The OpenTelemetry OpAMP Supervisor is intended to be the reference
+implementation of an OpAMP Supervisor, and as such will support all OpAMP
+capabilities. Additionally, it follows a design document for the features it
+intends to support.
+
+For a list of open issues related to the Supervisor, see [these issues](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues?q=is%3Aopen+is%3Aissue+label%3Acmd%2Fopampsupervisor).
+
+### OpAMP capabilities
+
+**Key**:
+
+✅: Fully implemented
+
+⚠️: Implemented with caveats
+
+📅: Planned, but no issue to track implementation
+
+| Status                                                                           | OpAMP capability               |
```

Comment:
Nit: I think it would bit a bit easier to read the table if the order of columns was reversed.

Reply:
That's a good point, done.